src/NServiceBus.AcceptanceTesting/Support/IConfigureTestExecution.cs

```diff
@@ -12,12 +12,12 @@ public interface IConfigureTestExecution
         /// <summary>
         /// 
         /// </summary>
-        /// <param name="configuration">The BusConfiguration instance.</param>
+        /// <param name="configuration">The EndpointConfiguration instance.</param>
```

Comment:
@danielmarbach sorry, github filtered the tag because I didn't format it as code